spec/src/main/resources/META-INF/smithy/bsp/extensions/jvm.smithy

```diff
@@ -150,4 +151,8 @@ list JvmCompileClasspathItems {
     member: JvmCompileClasspathItem
 }
 
-
+/// `JvmSourceItemData` contains jvm-specific metadata for a source item.
+@dataKind(kind: "jvm", extends: [SourceItemData])
+structure JvmSourceItemData {
+    packageName: String
```

Comment:
Please document it further. Is it supposed to always match the package declaration in the source file?